CI/azure/prepare_assets.sh

```diff
@@ -31,12 +31,20 @@ release_artifacts() {
                 rm -r "${i}"
         done
 
-        local zip_assets='2019 2022'
+        local zip_assets='VS-2019-x64 VS-2022-x64 MinGW-W64'
         cd "${BUILD_ARTIFACTSTAGINGDIRECTORY}"
+	mkdir Windows
+	cd Windows
+	mkdir include
+	cd ..
+	cp ./Windows-VS-2019-x64/iio.h ./Windows/include
         for i in $zip_assets; do
-		zip -r "Windows-VS-${i}-x64".zip "Windows-VS-${i}-x64"
-		rm -r "Windows-VS-${i}-x64"
+		rm ./"Windows-${i}"/iio.h
+		mv ./"Windows-${i}" Windows
         done
+	cp /home/vsts/work/1/s/CI/azure/zip.txt ./Windows
```

Comment:
we should rename this to "README.txt" in the zip file to make it obvious for people?

Reply:
sure, it can be renamed to README.txt